src/core_plugins/input_control_vis/public/vis.less

```diff
@@ -44,5 +44,9 @@ visualization.input_control_vis {
   }
 }
 
+visualize.input_control_vis {
+  padding: 0 !important;
```

Comment:
I think you should add a comment here explaining why this is important (I'm guessing bc of the width calculations?)